libnmstate/nm/applier.py

```diff
@@ -174,6 +179,9 @@ def set_ifaces_admin_state(ifaces_desired_state, con_profiles=()):
     for ifname in new_ifaces_to_activate:
         device.activate(dev=None, connection_id=ifname)
 
+    for ifname in new_vlan_ifaces_to_activate:
```

Comment:
These lines have to be moved below lines https://github.com/nmstate/nmstate/pull/464/files#diff-de1f01201a749955840472d0101bf589R194 - https://github.com/nmstate/nmstate/pull/464/files#diff-de1f01201a749955840472d0101bf589R196 , since the dummy interfaces **not** owned by NM are present within the `ifaces_to_edit` set. 

That fixes issue #461 , and unlocks the VLAN related test cases in vdsm.